app/config/variables.php

```diff
@@ -133,6 +133,14 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_REDIS_AUTH',
```

Comment:
We better keep consistency with other connector and use 2 vars instead of one. _APP_REDIS_USER and _APP_REDIS_PASS

Reply:
Ok!